pages/task/read/views/models/establishment.jsx

```diff
@@ -52,6 +56,21 @@ export default function Establishment({ task, values }) {
             proposedLabel={isComplete ? <Snippet>diff.changed-to</Snippet> : undefined}
           />
 
+          <div className="sticky-nav-anchor">
+            <h2><Snippet>conditions.title</Snippet></h2>
+            <Conditions
+              conditions={taskData.conditions ? taskData.conditions : establishment.conditions }
+              reminders={taskData.reminder !== '' ? [JSON.parse(taskData.reminder)] : establishment.reminders}
```

Comment:
Does this work if there are no reminders in the task?
Is the JSON.parse necessary?

Reply:
Have made the passing of reminders consistent
Parse is needed due to the way the data is passed through at the bottom of this file https://github.com/UKHomeOffice/asl-components/pull/277/files#diff-702fbff05fa84184acb68e5f199300abac056b3eabe558361ded0327cafce9f3

The object needs to be added as part of the form to be sent in the request when the amendment is accepted or returned
Happy to change it if there's an easier way